Form-Controls/index.html

```diff
@@ -1,27 +1,98 @@
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>My form exercise</title>
+  <meta name="description" content="Order a t-shirt from a selection of sizes and colours using this form" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+</head>
+
+<body>
+  <header>
+    <h1><u>T-Shirt Order Form</u></h1>
+  </header>
+  <main>
+    <form>
+
+
+      <br>
+      <label for="tshirt-size">Choose a T-shirt size: </label>
+      <br>
+      <select id="tshirt-size" name="tshirt-size" required
+        title="Please select a T-shirt size from the options in the list">
+        <option value="" disabled selected>-- Please choose --</option>
+        <option value="xs">X Small</option>
+        <option value="s">Small</option>
+        <option value="m">Medium</option>
+        <option value="l">Large</option>
+        <option value="xl">X Large</option>
+        <option value="xxl">XX Large</option>
+      </select>
+      <br>
+      <br>
+      <fieldset aria-labelledby="colour-label">
+        <legend id="colour-label">Choose a T-shirt Colour: </legend>
+        <label for="green">Green</label>
+        <input type="radio" name="colour" id="green" value="green" required>
+        <label for="black">Black</label>
+        <input type="radio" name="colour" id="black" value="black" required>
+        <label for="grey">Grey</label>
+        <input type="radio" name="colour" id="grey" value="grey" required>
+      </fieldset>
+
+      <br>
+      <br>
+      <fieldset>
+        <legend>Your Details</legend>
+        <p> Please complete <em>all</em> fields.</p>
+        
+        <label for="title">Title: </label>
+        <select id="title" name="title" required title="Please select your title from the options in the list">
+          <option value="" disabled selected>-- Please choose --</option>
+          <option value="mr">Mr.</option>
+          <option value="mrs">Mrs.</option>
+          <option value="ms">Ms</option>
+          <option value="mx">Mx</option>
+          <option value="miss">Miss</option>
+          <option value="dr">Dr.</option>
+          <option value="prof">Prof.</option>
+          <option value="rev">Rev.</option>
+        </select>
+        <br>
+        <br>
+        <label for="first-name">First Name: </label>
+        <input type="text" id="first-name" name="first-name" placeholder="e.g. Jane" required
+          pattern="[A-Za-z\s\-]{2,50}" title="Please enter your first name" aria-describedby="first-name-help">
+        <span id="first-name-help"><small>Please enter a valid first name using only letters and spaces, 2-50
```

Comment:
If you didn't want to pollute the user interface with this text all the time, you could probably use the `title` attribute of the input to make this information clear, but this works too :)